File: basic-calculator/cs/bcalculator.cs
using System;

class BCalculator{
    static void Main(){
        Console.WriteLine("Enter an integer: ");
        double num1;
        Double.TryParse(Console.ReadLine(), out num1);
        Console.WriteLine("Enter another integer: ");
        double num2;
        Double.TryParse(Console.ReadLine(), out num2);

        Console.WriteLine($"{num1} + {num2} = {num1 + num2}");
        Console.WriteLine($"{num1} - {num2} = {num1 - num2}");
        Console.WriteLine($"{num1} * {num2} = {num1 * num2}");
        Console.WriteLine($"{num1} / {num2} = {num1 / num2}");
    }

}

File: basic-calculator/lua/bcalculator.lua

function bcalculator()
    print("Enter an integer")
    local num1 = io.read("*n")
    print("Enter another integer")
    local num2 = io.read("*n")
    print(num1 .. " + " .. num2 .. " = " .. num1 + num2)
    print(num1 .. " - " .. num2 .. " = " .. num1 - num2)
    print(num1 .. " * " .. num2 .. " = " .. num1 * num2)
    print(num1 .. " / " .. num2 .. " = " .. num1 / num2)
end

bcalculator()